devnotes/scom_access.md

```diff
@@ -0,0 +1,92 @@
+# Access to SCOM registers
+
+According to information given on the OpenPower-firmware mailing list,
```

Comment:
Please provide a link to this conversation

Reply:
Added a link